gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
+#Wed Nov 27 15:07:39 CET 2019
+=======
```

Comment:
These are GitHub added text and will break the code

Reply:
Did you fetch this code?